static/js/services/rooms.js

```diff
@@ -135,6 +135,7 @@ define([
 				delete room.Credentials;
 			}
 			currentRoom = room;
+			$rootScope.roomType = room ? room.Type : null;
```

Comment:
None of the existing `room.updated` handlers are generic enough to store the type in a different scope, so I'll change this to listen for `room.updated` in the `uicontroller.js` and store the type in the scope there.